ws-test-harness/core/pom.xml

```diff
@@ -194,12 +196,13 @@
             <activation>
                 <property>
                     <!-- run tests using jaxws-ri -->
-                    <name>jaxws-image</name>
+                    <name>!metro-image</name>
```

Comment:
Is ! expected?

Reply:
yes, see the 'metro' profile, which relies on '<name>metro-image</name>'.

this particular change is just about changing the default runtime used for testing harness